### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Naylin15**
## Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Conventional Commit](http://conventionalcommits.org/)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

No content in the home page of the project
Issue # N/A

## What is the new behavior?
New content in the home page of the project

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Hi, @ajsb85 

This PR adds new content in the home page of the project

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- initsession()
+- killsession()
+- getMyProfiles()
+- getActiveProfile()
+- changeActiveProfile()
+- getMyEntities()
+- getActiveEntities()
+- changeActiveEntities()
+- getFullSession()
+- getAnItem()
```
Comment (ajsb85) [inline index.md]: The same old issue

**Review by ajsb85 — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (Naylin15) [reply index.md]: Fixing it

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Comment by Naylin15** [comment]
Changes made 👍

**Pull request merged**